gitlab_integration/README.md

```diff
@@ -0,0 +1,76 @@
+# Gitlab integration research
+
+## Gitlab Test Instance
+
+[http://52.183.132.26:3000/](http://52.183.132.26:3000/)
+
+### Crediantials
+
+- Username - testpackit
+- Password - Pass@123
```

Comment:
even though it's a test instance which I assume you can put down any time, you're putting your host in a terrible risk

Reply:
Yes, we can share the tokens in a private way.

Reply:
I had put many restrictions before sharing it here. But yeah makes sense.